examples/example.php

```diff
@@ -56,7 +56,7 @@
 $header->setSendAt(1409348513); # must be a unix timestamp
 
 # This gives you back the stringified json formatted X-SMTPAPI header. Use this with your [smtp client](https://github.com/andris9/simplesmtp) of choice.
-print $header->jsonString();
+print_r $header->jsonString();
```

Comment:
How about using the `var_dump` function? It's more detailed information than using `print_r` function.